src/amo/components/AutoSearchInput/index.js

```diff
@@ -134,7 +131,7 @@ export class AutoSearchInputBase extends React.Component<InternalProps, State> {
   }
 
   createFiltersFromQuery(query: string) {
-    const { location, userAgentInfo } = this.props;
```

Comment:
Looks like we're not using `userAgentInfo` anymore so we could remove it fully.

Reply:
0d9491a5e2be5e0cc73c61b792312b30b2137067